Sources/Fullscreen/Explore/ViewModels/ExploreRecommendationAdViewModel.swift

```diff
@@ -0,0 +1,43 @@
+import FinniversKit
+
+final public class ExploreRecommendationAdViewModel: StandardAdRecommendationViewModel, Hashable {
```

Comment:
I see plenty of data types are public. Do they have to be? Are the views and models used "outside" the app somewhere?

Reply:
Sorry, total confusion on my part. I thought I was reading from the app repo. 😆 Yes, public is needed for demo app.

Reply:
No worries, I can't relate at all, I'm not confused at all working in 4 different repos on this stuff 😂 🙈